src/libraries/System.Text.Json/tests/Serialization/NumberHandlingTests.cs

```diff
@@ -1619,5 +1621,108 @@ public static void JsonNumberHandling_ArgOutOfRangeFail()
             Assert.Throws<ArgumentOutOfRangeException>(
                 () => new JsonNumberHandlingAttribute((JsonNumberHandling)(8)));
         }
+
```

Comment:
Are there existing non-Stream tests that should be converted to use the `DeserializationWrapper` pattern for Streams? Instead of authoring new tests?

Reply:
We didn't have coverage for this scenario for both stream and non-stream cases. The issue this PR fixes applies to the non-stream (sync) case. I've adapted the tests to use the `DeserializationWrapper` pattern.